examples/data-solutions/data-playground/outputs.tf

```diff
@@ -14,12 +14,15 @@
 
```

Comment:
Would it make sense to include the created dataset as well in the output. Might be useful.

Reply:
Done